macros/plotClusterEvent.C

```diff
@@ -0,0 +1,128 @@
+#include <TTree.h>
+#include <TFile.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <Rtypes.h>
+#include <TCanvas.h>
+
+#include <cstdlib>
+#include <iostream>
+#include <iomanip>
+#include <unistd.h>
+
+#include "CTSEvent.h"
+#include "CTSEventClusters.h"
+#include "Utility.h"
+
+///< usage: ./plotClusterEvent -i inputfile -o outputfile -n numberOfEventsToBeProcessed
+///< n = -1 by default which means the whole file is processed
+
+extern char* optarg;
+
+void plotClusterEvent(const char *inputFile, const char *outputFile, ULong_t procNr)
```

Comment:
Here I would prefer `plotCTSEventClusters` and I see that the name of the cluster event class was poorly chosen by me. :smile:.